test/distributed/org/apache/cassandra/fuzz/ring/ConsistentBootstrapTest.java

```diff
@@ -219,15 +221,17 @@ public void coordinatorIsBehindTest() throws Throwable
                     {
                         if (cluster.get(1).config().broadcastAddress().toString().contains(replica.node().id()))
                         {
-                            HistoryBuilderHelper.insertRandomData(schema, i, ckGen.generate(rng), rng, history);
-                            break outer;
+                            try
+                            {
+                                HistoryBuilderHelper.insertRandomData(schema, i, ckGen.generate(rng), rng, history);
```

Comment:
why was the `break outer;` removed?

Reply:
The problem was that the test was just implemented in a wrong way. I did my best to reproduce pre-harry-2 logic. When rewriting the patch for harry2, rather than catching-and-ignoring the exception, and then looking at the log, the test was just throwing an exception, and then timing out because one of the nodes was paused.